src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -77,11 +78,37 @@ public DefaultCaseDefinitionRepository(final ApplicationParams applicationParams
         }
     }
 
+    @Override
+    public List<String> getCaseTypesReferences() {
+        return getCaseTypesReferences(null);
+    }
+
+    public List<String> getCaseTypesReferences(final HttpHeaders httpHeaders) {
```

Comment:
Can we declare `httpHeaders` Optional? rather than passing a null?

Reply:
Optional as method parameters? It is advised to use it as a return type only. Please see:
http://dolszewski.com/java/java-optional-parameters/
3.3. Why not Optional as method argument?

Reply:
As the guys says, this is very controversial. I see some of his points. You can leave it as you prefer